### PR TITLE
012

### PR DESCRIPTION
**Pull request by bstrie**

### CODE REVIEW

File: _episodes/012-pietro-pascal-santiago.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "RustFest Interviews Triple Feature: Rust Release Engineering; Developing the Developer Tools; Rust in Latin America"
+date: 2015-05-15T16:00:00Z
```
Comment (jonhoo) [inline _episodes/012-pietro-pascal-santiago.md]: This seems wrong

File: _episodes/012-pietro-pascal-santiago.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- [@18:23] - What is the Developer Tools Team?
+- [@19:39] - What tools is the Developer Tools Team responsible for, and what purposes do they serve?
+- [@24:46] - Which tools in particular would you like to draw attention to?
+- [@26:19] - How does rust-analyzer compare to RLS?
+- [@29:42] - How does the Developer Tools Team coordinate?
+- [@32:00] - How was your experience at RustFest this year?
+
+#### [@36:21] Part 3: Rust Latam w/ Santiago Pastorino
+
+- [@36:46] - What is Rust Latam?
```
Comment (jonhoo) [inline _episodes/012-pietro-pascal-santiago.md]: Maybe add a link to Rust Latam?

File: _episodes/012-pietro-pascal-santiago.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+### Timestamps & referenced resources
+
+#### [@00:00] Part 1: Crater & Rust Release Infrastructure w/ Pietro Albini
+
+- [@01:01] - What is your role in the Rust project?
+- [@01:46] - What lessons did the infrastructure team learn from the Rust 2018 release?
+- [@03:29] - How do you feel about potential future Rust editions in 2021 or beyond?
+- [@06:26] - Do you think Rust's regular release cycle too fast or too slow?
+- [@08:56] - How does Crater guard against language regressions, and what things doesn't it catch?
```
Comment (jonhoo) [inline _episodes/012-pietro-pascal-santiago.md]: Maybe a link to crater?

File: _episodes/012-pietro-pascal-santiago.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- [@11:12] - How has Crater scaled as the ecosystem has grown, and is it at risk of becoming infeasible to run?
+- [@16:17] - How can someone get involved with the Infrastructure Team?
+
+#### [@17:25] Part 2: Developer Tools w/ Pascal Hertleif
+
+- [@18:23] - What is the Developer Tools Team?
+- [@19:39] - What tools is the Developer Tools Team responsible for, and what purposes do they serve?
+- [@24:46] - Which tools in particular would you like to draw attention to?
+- [@26:19] - How does rust-analyzer compare to RLS?
```
Comment (jonhoo) [inline _episodes/012-pietro-pascal-santiago.md]: Link to rls and link to rust-analyzer?

File: _episodes/012-pietro-pascal-santiago.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+#### [@00:00] Part 1: Crater & Rust Release Infrastructure w/ Pietro Albini
+
+- [@01:01] - What is your role in the Rust project?
+- [@01:46] - What lessons did the infrastructure team learn from the Rust 2018 release?
+- [@03:29] - How do you feel about potential future Rust editions in 2021 or beyond?
+- [@06:26] - Do you think Rust's regular release cycle too fast or too slow?
+- [@08:56] - How does Crater guard against language regressions, and what things doesn't it catch?
+- [@11:12] - How has Crater scaled as the ecosystem has grown, and is it at risk of becoming infeasible to run?
+- [@16:17] - How can someone get involved with the Infrastructure Team?
```
Comment (jonhoo) [inline _episodes/012-pietro-pascal-santiago.md]: Relevant links?

File: _episodes/012-pietro-pascal-santiago.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+#### [@36:21] Part 3: Rust Latam w/ Santiago Pastorino
+
+- [@36:46] - What is Rust Latam?
+- [@37:42] - What inspired you to start a Rust conference in Latin America?
+- [@39:06] - How big is Rust Latam?
+- [@40:15] - What is interest in Rust like in Latin America?
+- [@42:42] - What is the broader software industry like in Latin America?
+- [@44:59] - What's next for Rust Latam?
+- [@45:42] - How did you get into Rust?
+- [@50:17] - What venues are there for Spanish or Portuguese-speaking Rust users?
```
Comment (jonhoo) [inline _episodes/012-pietro-pascal-santiago.md]: Relevant links?

File: _episodes/012-pietro-pascal-santiago.md (hunk 1 above)
Comment (bstrie) [reply _episodes/012-pietro-pascal-santiago.md]: Yeah I'm instituting a policy of not bothering to figure out the date until just before I'm about to publish. :P

File: _episodes/012-pietro-pascal-santiago.md (hunk 6 above)
Comment (bstrie) [reply _episodes/012-pietro-pascal-santiago.md]: I didn't actually get any links, mostly it's just "yeah they exist and they're in telegram"

**Pull request merged**